src/app/features/public-form/edit-public-form-columns/edit-public-form-columns.component.ts

```diff
@@ -44,7 +44,9 @@ export class EditPublicFormColumnsComponent
 
   updateValue(newConfig: FormConfig) {
     // setTimeout needed for change detection of disabling tabs
-    setTimeout(() => this.formControl.setValue(newConfig.fieldGroups));
+    setTimeout(() =>
+      this.formControl.setValue(newConfig.fieldGroups, { emitEvent: false }),
```

Comment:
will this not make it impossible for entity forms or other parent components to subscribe to changes of this, if it should be necessary?

Reply:
I debugged this again and found the issue: every time any value changes in the form (even typing a single letter in the title), we update this.formConfig, which is passed as an @Input to admin-entity-form. This re-triggers change detection, causing the input to re-render and focus being lost